examples/cli/elan/train.py

```diff
@@ -48,7 +48,8 @@
 # Link dataset and pd to a new model, then train the model.
 if 'mx' not in elpis.list_models():
     m = elpis.new_model('mx')
-    m.link(ds, pd)
+    m.link_dataset(ds)
+    m.link_pron_dict(pd)
```

Comment:
This is fine. Please reset the `tx` change above and I'm happy for this to be merged - it copies the behaviour of the model new endpoint.